roles/core/log_client/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+log_client_rsyslog_port: 514
```

Comment:
Maybe call it log_client_rsyslog_server_port ? To explain this is the server port, and not the client port ?

Reply:
well done... :-) ... did not see the last line of the template.
It is all fixed ... hopefully.